pom.xml

```diff
@@ -79,6 +79,17 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Instead of using these dependencies for sending emails, consider to use `spring-boot-starter-mail`.

The XML to add it as a dependency is the following one:

```xml
<dependency>
    <groupId>org.springframework.boot</groupId>
	<artifactId>spring-boot-starter-mail</artifactId>
</dependency>
```